docs/developer-guide/working-with-rules.md

```diff
@@ -397,10 +397,10 @@ The rule naming conventions for ESLint are fairly simple:
 
 ## Rule Acceptance Criteria
 
-Because rules are highly personal (and therefore very contentious), the following guidelines determine whether or not a rule is accepted and whether or not it is on by default:
+Because rules are highly personal (and therefore very contentious), the following guidelines determine whether or not a rule is accepted:
 
-* If the same rule exists in JSHint and is turned on by default, it must have the same message and be enabled by default.
-* If the same rule exists in JSLint but not in JSHint, it must have the same message and be disabled by default.
+* If the same rule exists in JSHint and is turned on by default, it must have the same message and be recommended.
```

Comment:
I think we should remove this section now. We can't make any guarantees about JSHint/JSLint compat.